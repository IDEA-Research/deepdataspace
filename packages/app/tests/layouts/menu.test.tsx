import Menu from '@/layouts/menu';
import { umiRender } from '../test-utils';

test('render Menu by snapshot', () => {
  const { container } = umiRender(<Menu collapsed={false} />);
  expect(container).toMatchSnapshot();
});
