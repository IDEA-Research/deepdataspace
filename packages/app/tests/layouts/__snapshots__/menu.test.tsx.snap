// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Menu by snapshot 1`] = `
<div>
  <ul
    class="ant-menu ant-menu-root ant-menu-inline ant-menu-light menu css-dev-only-do-not-override-zjzpde"
    data-menu-list="true"
    dir="ltr"
    role="menu"
    tabindex="0"
  >
    <li
      class="ant-menu-item"
      data-menu-id="rc-menu-uuid-test-Dataset"
      link="/dataset"
      role="menuitem"
      style="padding-left: 24px;"
      tabindex="-1"
      title=""
    >
      <span
        aria-label="database"
        class="anticon anticon-database ant-menu-item-icon"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="database"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M832 64H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V96c0-17.7-14.3-32-32-32zm-600 72h560v208H232V136zm560 480H232V408h560v208zm0 272H232V680h560v208zM304 240a40 40 0 1080 0 40 40 0 10-80 0zm0 272a40 40 0 1080 0 40 40 0 10-80 0zm0 272a40 40 0 1080 0 40 40 0 10-80 0z"
          />
        </svg>
      </span>
      <span
        class="ant-menu-title-content"
      >
        Dataset
      </span>
    </li>
    <li
      class="ant-menu-item"
      data-menu-id="rc-menu-uuid-test-Annotate"
      link="/project"
      role="menuitem"
      style="padding-left: 24px;"
      tabindex="-1"
      title=""
    >
      <span
        aria-label="gateway"
        class="anticon anticon-gateway ant-menu-item-icon"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="gateway"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M928 392c8.8 0 16-7.2 16-16V192c0-8.8-7.2-16-16-16H744c-8.8 0-16 7.2-16 16v56H296v-56c0-8.8-7.2-16-16-16H96c-8.8 0-16 7.2-16 16v184c0 8.8 7.2 16 16 16h56v240H96c-8.8 0-16 7.2-16 16v184c0 8.8 7.2 16 16 16h184c8.8 0 16-7.2 16-16v-56h432v56c0 8.8 7.2 16 16 16h184c8.8 0 16-7.2 16-16V648c0-8.8-7.2-16-16-16h-56V392h56zM792 240h88v88h-88v-88zm-648 88v-88h88v88h-88zm88 456h-88v-88h88v88zm648-88v88h-88v-88h88zm-80-64h-56c-8.8 0-16 7.2-16 16v56H296v-56c0-8.8-7.2-16-16-16h-56V392h56c8.8 0 16-7.2 16-16v-56h432v56c0 8.8 7.2 16 16 16h56v240z"
          />
        </svg>
      </span>
      <span
        class="ant-menu-title-content"
      >
        Annotate
      </span>
    </li>
    <li
      class="ant-menu-item"
      data-menu-id="rc-menu-uuid-test-Lab"
      link="/lab"
      role="menuitem"
      style="padding-left: 24px;"
      tabindex="-1"
      title=""
    >
      <span
        aria-label="bulb"
        class="anticon anticon-bulb ant-menu-item-icon"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="bulb"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M632 888H392c-4.4 0-8 3.6-8 8v32c0 17.7 14.3 32 32 32h192c17.7 0 32-14.3 32-32v-32c0-4.4-3.6-8-8-8zM512 64c-181.1 0-328 146.9-328 328 0 121.4 66 227.4 164 284.1V792c0 17.7 14.3 32 32 32h264c17.7 0 32-14.3 32-32V676.1c98-56.7 164-162.7 164-284.1 0-181.1-146.9-328-328-328zm127.9 549.8L604 634.6V752H420V634.6l-35.9-20.8C305.4 568.3 256 484.5 256 392c0-141.4 114.6-256 256-256s256 114.6 256 256c0 92.5-49.4 176.3-128.1 221.8z"
          />
        </svg>
      </span>
      <span
        class="ant-menu-title-content"
      >
        Lab
      </span>
    </li>
  </ul>
  <div
    aria-hidden="true"
    style="display: none;"
  />
</div>
`;
